### PR TITLE
Updated dependencies to remove suppressed CVEs

### PR DESCRIPTION
**Pull request by colmlynch60**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RIA-6892

### Change description ###

Updates to address CVEs

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by kalachandrasekar1 — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/iacaseaccessapi/infrastructure/config/SwaggerConfiguration.java [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/*
```
Comment (kalachandrasekar1) [inline src/main/java/uk/gov/hmcts/reform/iacaseaccessapi/infrastructure/config/SwaggerConfiguration.java:1]: @colmlynch60 Any reason to comment out this entire file

Comment (colmlynch60) [reply src/main/java/uk/gov/hmcts/reform/iacaseaccessapi/infrastructure/config/SwaggerConfiguration.java]: Apologies - I removed it when upgrading swagger dependency. I've replaced this configuration file with OpenApiConfiguration and same as other IAC services.

**Review by javimarinc96 — APPROVED** [review]
LGTM

**Review by filippo-possenti-version1 — APPROVED** [review]

**Review by rajeshthuraiyur — APPROVED** [review]
LGTM

**Pull request merged**